<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="cloudodyssey" failures="1" tests="1" name="Test" time="2.688" errors="0" timestamp="2024-12-19T16:20:48 IST">
  <testcase classname="testCases.TC002_LoginDDT" name="verify_loginDDT" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.io.FileNotFoundException: .\testData\Login_TestData2.xlsx (The system cannot find the file specified)">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: .\testData\Login_TestData2.xlsx (The system cannot find the file specified)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.io.FileNotFoundException: .\testData\Login_TestData2.xlsx (The system cannot find the file specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at utilities.ExcelUtility.getRowCount(ExcelUtility.java:31)
at utilities.DataProviders.getData(DataProviders.java:19)
... 25 more
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
